src/token/types.ts

```diff
@@ -1 +1,19 @@
 // 🌲☀️🌲
+
+import { Address } from 'viem';
+
+export type Token = {
+  name: string;
+  currencyCode: string;
+  imageURL: string;
+  blockchain: string;
```

Comment:
I don't think we need this. As OnchainKit is mostly meant for ETH at this point.